src/server/debugging.ts

```diff
@@ -53,6 +54,7 @@ type RawProcessInfo = {
     containerPID: number;
     containerPPID: number;
     containerPGID: number;
+    runState: string;
```

Comment:
As you've figured out, there are multiple stages to parsing the output of the kernel module. First, there is local parsing/validation (e.g. is the pid a number? it should be!) into Raw*Info objects, and then there is global parsing/validation, where we reorganize all the data.

Since parsing the runState can be done locally (without knowledge of any other processes), I think it's better to do so early, and store this as a `ProcessRunState` instead of a `string`.